mimic/model/rackspace_images.py

```diff
@@ -98,6 +98,83 @@ def detailed_json(self, absolutize_url):
         return template
 
 
+@attributes(['image_id', 'tenant_id', 'name', 'minDisk', 'minRam', 'image_size', 'server_id', 'links',
+             'flavor_classes', 'os_type', 'os_distro', 'vm_mode', 'disk_config'])
+class RackspaceSavedImage(object):
+    """
+    A Rackspace saved image object representation
```

Comment:
This class looks a *lot* like `mimic.model.glance_objects.Image`.  How are they different?  If we add this class it seems like a lot of duplicated code to generate very similar JSON data structures...